sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md

```diff
@@ -3,26 +3,19 @@
 ## Key concepts
 This sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts only.
 
-1. The bearer token is obtained from the request header. 
+1. The bearer token is obtained from the request header.
 2. `JwtDecoder`  is used to parse the token into `Jwt`.
-3. Claims, headers etc in `Jwt` will be extracted, they will be wrapped in `AzureOAuth2AuthenticatedPrincipal` object.
-4. `AzureOAuth2AuthenticatedPrincipal` will eventually be set into SecurityContext.
-
+3. Verify `aud`, `iss`, `nbf`, `exp` in access token.
+4. After successful verification, claims, headers etc in `Jwt` will be extracted, they will be wrapped in `AzureOAuth2AuthenticatedPrincipal` object.
```

Comment:
We've changed this class's name.

Reply:
How about change this sentence to `Extract information from JWT in xxx object after a successful verification` ?

Reply:
I've updated these directly. Resolve this conversation now.